test/db/cmd/cmd_print

```diff
@@ -747,7 +747,7 @@ CMDS=<<EOF
 . scripts/palette.r2
 woe 0 0xff 1
 (test_flag; f `p8 1`)
-.(test_flag) @@s:0 0x100 1
```

Comment:
mmm @thestr4ng3r @XVilka what do you think about this? I'm actually used, in general, to have [start, end) behaviour about loops. Thought it is also true that this way you can't iterate up to `UT64_MAX`.

Reply:
@ret2libc Could you clarify what you meant by:

https://github.com/radareorg/radare2/blob/3d2f02be246999bb98190559029bbce21bdd4b28/shlr/radare2-shell-parser/corpus/iter_commands.txt#L164

?

Reply:
It's also true that `woe` uses inclusive limits :/

Reply:
> @ret2libc Could you clarify what you meant by:
> 
> https://github.com/radareorg/radare2/blob/3d2f02be246999bb98190559029bbce21bdd4b28/shlr/radare2-shell-parser/corpus/iter_commands.txt#L164
> 
> ?

What do you mean?
That is a tree-sitter test to make sure the newshell is able to parse 
`x @@s:from to step  run 'x' on all offsets from, to incrementing by step`.

So in that case it would run `p8` on `0xd000`, `0xd100`, .... up to 0xe000 (excluded in master, included in your pr)

Reply:
> So in that case it would run `p8` on `0xd000`, `0xd100`, .... up to 0xe000 (excluded in master, included in your pr)

So if I wanted to go from `0xd000` to `0xe000` inclusive, I would have to do:

```
p8 @@s:0xd000 0xe001 0x100
```
?

Reply:
right now yes.. I don't have super strong opinions on which way is better

Reply:
> I'm actually used, in general, to have [start, end) behaviour about loops.

This is not a for loop that we are talking about, it's a range from a to b.

Consider the range `['A', 'Z']`. How are you going to represent it using [start, end)? By using `['A', '[')`? I certainly didn't remember that '[' comes after 'Z'.

> Though it is also true that this way you can't iterate up to UT64_MAX.

I understand that this is pedantic and that possibly nobody is going to need to iterate up to UT64_MAX. But are you 100% sure? What happens if suddenly there is a need for it?

> I don't have super strong opinions on which way is better

The help message says:

```
| x @@s:from to step  run 'x' on all offsets from, to incrementing by step
```

**all offsets** ... please bring the code in line with the help message.

Reply:
Yeah I guess you were convincing enough :D